core/tests/Drupal/Nightwatch/env.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+
+const env = {};
+
+if (!fs.fileExistsSync('../../../nightwatch.settings.json')) {
```

Comment:
should be fs.existsSync('nightwatch.settings.json') -> process.cwd() says its in /var/www/html/core already, and is not relative to the directory that env.js exists in.